.vscode/extensions.json

```diff
@@ -1,9 +0,0 @@
-{
-  "recommendations": [
```

Comment:
Why was this deleted?

Reply:
did not mean to commit that, ill add it back